[2022-07-28 15:09:37,587] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NVDA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:37,649] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: NVDA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:37,657] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:37,661] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-28 15:09:37,661] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:37,700] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-28 14:30:00+00:00
[2022-07-28 15:09:37,705] {standard_task_runner.py:52} INFO - Started process 804 to run task
[2022-07-28 15:09:37,741] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'NVDA_historic_data', 'create_historic_table', 'scheduled__2022-07-28T14:30:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmpoujct1z0', '--error-file', '/tmp/tmpp9k1d5qu']
[2022-07-28 15:09:37,752] {standard_task_runner.py:80} INFO - Job 100: Subtask create_historic_table
[2022-07-28 15:09:38,201] {task_command.py:371} INFO - Running <TaskInstance: NVDA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [running]> on host a2299eb83309
[2022-07-28 15:09:40,202] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=NVDA_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T14:30:00+00:00
[2022-07-28 15:09:40,369] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-28 15:09:40,387] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__NVDA_ (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            symbol VARCHAR(32) NOT NULL,
            PRIMARY KEY (symbol, datetime)
        )
    , parameters: None
[2022-07-28 15:09:40,458] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 15:09:40,517] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=NVDA_historic_data, task_id=create_historic_table, execution_date=20220728T143000, start_date=20220728T150937, end_date=20220728T150940
[2022-07-28 15:09:40,618] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 15:09:41,320] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
